app/server/schemas.py

```diff
@@ -153,6 +153,8 @@ def filter_rejected(self, data, many):
 
     transfer_metadata       = fields.Function(lambda obj: obj.transfer_metadata)
 
+    token                   = fields.Nested(TokenSchema, only=('id', 'symbol'))
```

Comment:
this might cause tests to fail depending on what checks are on the schemas

Reply:
hmmm @enjeyw how would this work for credit transfers from one token to another and exchanges?

Reply:
Exchanges are necessarily made of two credit transfers, one for each token type, so this _should_ be fine I think